flink-runtime/src/test/java/org/apache/flink/runtime/dispatcher/TestingDispatcher.java

```diff
@@ -80,9 +80,16 @@ void completeJobExecution(ArchivedExecutionGraph archivedExecutionGraph) {
 	}
 
 	@VisibleForTesting
-	public CompletableFuture<Void> getJobTerminationFuture(@Nonnull JobID jobId, @Nonnull Time timeout) {
+	CompletableFuture<Void> getJobTerminationFuture(@Nonnull JobID jobId, @Nonnull Time timeout) {
 		return callAsyncWithoutFencing(
 			() -> getJobTerminationFuture(jobId),
 			timeout).thenCompose(Function.identity());
 	}
+
+	@VisibleForTesting
```

Comment:
Not sure about `@VisibleForTesting` here. This class is already a test utility. It is even in the `test` sources directory.

Reply:
You're right. Will change it.